ruby/introduction/installing_ruby.md

```diff
@@ -106,7 +106,27 @@ rbenv install 3.1.2 --verbose
 
 This command will take 10-15 minutes to complete. The `--verbose` flag will show you what's going on so you can be sure it hasn't gotten stuck. While it installs, take this time to watch [this video](https://youtu.be/X2CYWg9-2N0) or to get a glass of water.
 
-When the last command is finished, set the Ruby version and verify that it's working:
+#### Troubleshooting
```

Comment:
One issue with this subheading is now the rest of this section is nested under it, and it seems like all the steps are about troubleshooting. I think this either needs to go to the end of the section, or another subheading needs to be added. I think my preference would be the first, but either could probably work.

An alternative would be to simply make updating `ruby-build` a part of the steps, as it's a lot less verbose overall. What do you think about these options?

Reply:
> I think this either needs to go to the end of the section

I like the idea of placing `#### Troubleshooting` at the end of the section, but:

> An alternative would be to simply make updating `ruby-build` a part of the steps, as it's a lot less verbose overall.

I like this a bit more because the hooks/step subheadings are giving me a sense of comfort and consistency. Maybe that extends to others as well.

I could also discard chunking it as a step and integrate it into `#### Step 2.2: Install Ruby` (without a `#### Troubleshooting` subheading or step). That's likely what you were implying as well. So I'll go with this one!